packages/server/graphql/mutations/editReflection.ts

```diff
@@ -35,7 +35,7 @@ export default {
 
     // AUTH
     const phaseItem = await r
```

Comment:
`phaseItem` can be renamed to `reflectPrompt` to keep names meaningful

Reply:
should i rename the phaseItemId variable as well?

Reply:
yep!